### DISCUSSION TITLE
Merge squash

### DISCUSSION THREAD

**Discussion by mestihudson** [jesseduffield/lazygit#1296] [Q&A] [answered]
Is there a way to get a `git merge <branch> --squash`?
I know that is possible to use a custom command, but I'd like to follow the lazygit way.

Thanks for this awesome tool.

**Comment by jesseduffield** [comment, marked as answer]
That's currently not possible, but it makes sense. I recommend creating an issue for this if it doesn't already exist :)

**Comment by lougreenwood** [comment]
@mestihudson Did you manage to create an issue for this?

**Reply by mestihudson** [reply]
No. I forgot. Sorry.

**Reply by mark2185** [reply]
[Here.](https://github.com/jesseduffield/lazygit/issues/2028)
